webpack.config.js

```diff
@@ -124,9 +121,19 @@ module.exports = {
     ],
   },
   plugins: [
-    new MiniCssExtractPlugin({
-      filename: 'css/[name].css',
+    new HtmlBundlerPlugin({
```

Comment:
so simple and easily you can configure the rendering templates containing source assets into static HTML